src/main/java/frc/robot/subsystems/intake/Intake.java

```diff
@@ -11,11 +11,21 @@ public Intake(IntakeIO io) {
     this.io = io;
   }
 
+  /**
+   * sets the angle of the pivot in rotations
+   *
+   * @param angle desired angle in rotations
+   */
   public void setPivotAngle(double angle) {
```

Comment:
id add a conversion from degrees to rotations here to make it easier to tune. Also capitilize stuff